Integrations/integration-VirusTotal-Private_API.yml

```diff
@@ -651,14 +663,21 @@ script:
         return http_request('GET', api_endpoint, params, DEFAULT_HEADERS)
 
 
+    def check_detected_urls_threshold(detected_urls, threshold):
+        for url in detected_urls:
+            if url.get("positives") > threshold:
```

Comment:
By the description of the threshold param, I think it makes sense for the operator to be `>=`

Reply:
Fixed